### PR TITLE
Add documentation comments for volume expand controller

### PR DESCRIPTION
**Pull request by gnufied**
These comments help clear out some of the design choices made
in code.

cc @kubernetes/sig-storage-pr-reviews @NickrenREN

### CODE REVIEW

File: pkg/controller/volume/expand/pvc_populator.go [hunk 1]
```diff
@@ -80,6 +80,7 @@ func (populator *pvcPopulator) Sync() {
 			glog.V(5).Infof("Error getting persistent volume for pvc %q : %v", pvc.UID, err)
 			continue
 		}
+		// We are only going to add PVCs which are bound and for which pvc.Spec.Size > pvc.Status.Size
```
Comment (NickrenREN) [inline pkg/controller/volume/expand/pvc_populator.go]: Can we clarify here that the check `pvc.Spec.Size > pvc.Status.Size` is preformed by `AddPVCUpdate`

Comment (gnufied) [reply pkg/controller/volume/expand/pvc_populator.go]: yeah sure.

**Comment by gnufied** [comment]
@jsafrane it probably needs /approve no-issue

**Pull request merged**